pkg/apiserver/proxy.go

```diff
@@ -56,6 +56,7 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	var apiResource string
 	var httpCode int
 	reqStart := time.Now()
+	pretty := isPrettyPrint(req)
```

Comment:
Yeah, I think that makes the most sense.  Until we have larger errors (I have a todo to make an update conflict return the conflicting resource) we can get away with always indenting errors for a lot of code.

> On Jun 6, 2015, at 12:14 PM, Brian Grant <notifications@github.com> wrote:
> 
> In pkg/apiserver/proxy.go:
> 
> > @@ -56,6 +56,7 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
> >  	var apiResource string
> >  	var httpCode int
> >  	reqStart := time.Now()
> > +	pretty := isPrettyPrint(req)
> Or we could just make indentation true by default for proxy errors.
> 
> —
> Reply to this email directly or view it on GitHub.
>